src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.searchbar {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+}
+
+.input__field {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.big--searchbar {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.big--search__icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small--searchbar {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.small--search__icon {
```

Comment:
instead of creating .big--class, .small--class you can create one base class, for example search-bar where you will add all needed styles to the big input, and you can create one class where you will override those styles for the small input, search-bar--small